lib/Patat/Images/WezTerm.hs

```diff
@@ -42,20 +70,63 @@ new config = do
 drawImage :: FilePath -> IO Cleanup
 drawImage path = do
     content <- B.readFile path
+
+    wez <- wezExecutable
+    resp <- fmap BLU.fromString $ readProcess wez ["cli", "list", "--format", "json"] []
```

Comment:
Use `Data.Text.Encoding.encodeUtf8 . Data.Text.pack` to avoid the `utf8-string` dependency?